tests/unit/workflows/deploy.spec.ts

```
@@ -0,0 +1,324 @@
+  test('deployment script should use PM2 directly', async ({ assert }) => {
+    // The updated workflow uses pm2 directly instead of pnpx pm2
+    assert.match(workflowContent, /pm2 restart 0/, 'Should restart PM2 process 0')
+    assert.notMatch(workflowContent, /pnpx pm2 restart/, 'Should not use pnpx for pm2 restart')
+    
+    assert.match(workflowContent, /pm2 save/, 'Should save PM2 configuration')
+    assert.notMatch(workflowContent, /pnpx pm2 save/, 'Should not use pnpx for pm2 save')
+  })
+
+  test('deployment script should have proper command ordering', async ({ assert }) => {
+    // Extract the script section
+    const scriptMatch = workflowContent.match(/script:\s*\|([^]*?)(?=\n\s{0,10}\S|$)/m)
```

Biome (2.1.2)

[error] The regular expression includes this negated empty character class.

Negated empty character classes match anything.
If you want to match against [, escape it \[.
Otherwise, remove the character class or fill it.

(lint/correctness/noEmptyCharacterClassInRegex)